meta-lmp-bsp/dynamic-layers/meta-ti-bsp/recipes-support/ti-mfgtool-files/ti-mfgtool-files/flash.sh.in

```diff
@@ -131,40 +131,63 @@ fi
 [ -z "${boardType}" ] && boardType="gp"
 [ -z "${devID}" ] && devID="$(lsusb | grep 'Texas Instruments, Inc. AM62x' | awk '{print $6}')"
 
+vid=$(echo "${devID}" | cut -d ':' -f 1)
+pid=$(echo "${devID}" | cut -d ':' -f 2)
+
 ####
 # Flash software images
 ####
 
 # start USB DFU download mode and flash primary eMMC with provided WIC image
 
 echo "Load U-Boot via DFU..."
+dfu-suffix -c ${scriptDir}/tiboot3-am62x-${boardType}-evm.bin || dfuCheck=$?
```

Comment:
Using the || will mean that dfuCheck is only set if return is non zero and I do not see an initialization of dfuCheck

Reply:
fixed